CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+9.0.1
+-----
+
+* Minor docs corrections
```

Comment:
I think this should be `doc` singular (since the abbreviation is documentation, not documentations)